inference-engine/tests/functional/plugin/gpu/single_layer_tests/loop.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <tuple>
+#include <string>
+#include <vector>
+#include <memory>
+#include "single_layer_tests/loop.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+using namespace LayerTestsDefinitions;
+using namespace InferenceEngine;
+
+namespace {
+    static const std::vector<std::tuple<bool, int64_t, int64_t, int64_t>> static_loop_types {
+            //  GCC4.8 limitation: have to specify type of each element in list
+            //                               static_trip_count |  max | dynamic_exit | axis
+            std::tuple<bool, int64_t, int64_t, int64_t>{  true ,  5, -1, -1 },  // n_iter 5, no dynamic exit
+        //     std::tuple<bool, int64_t, int64_t, int64_t>{  true ,  5,  3, -1 },  // n_iter 3, dynamic exit on 3
+        //     std::tuple<bool, int64_t, int64_t, int64_t>{  true ,  5,  7, -1 },  // n_iter 5, dynamic exit not reached
+        //     std::tuple<bool, int64_t, int64_t, int64_t>{  true , -1,  5, -1 },  // n_iter 5, inf loop with dynamic exit on 5
```

Comment:
@EunsooSheenIntel  Is there specific reason that you disabled these shapes?

Reply:
The testcases which is commented out are dynamic exit loop.